src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -130,16 +133,22 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
 
   //
   // Exclude the first '/' from the Sub Service
+  // But, only if it starts with a '/'
   //
-  const char* subService = &subServ.c_str()[1];
+  const char* subService = subServ.c_str();
+
+  if (subService[0] == '/')
+  {
+    subService = &subService[1];
+  }
```

Comment:
Yes, I was thinking about that too.
However, not sure it's a good idea to cancel the cause of the error, which in this case in an erroneous service path ... ?

We would need a special keyword, but, the info about the erroneous service path goes away. That's a pity ...

I'll include this in the ToDo in the header file as a discussion item. Perhaps we can find a better solution